### PR TITLE
added  commands

### PR DESCRIPTION
**Pull request by RAJU2529**
added commands after verification,  i tested on my laptop.

### CODE REVIEW

**Comment by RAJU2529** [comment]
@illfated  @jvsam @e0i . Please check the added commands, if you found is correct then approve it

Thanking you

**Comment by RAJU2529** [comment]
@opbld33 .So now what should i do

File: docset/windows/dism/Mount-WindowsImage.md [hunk 1]
```diff
@@ -95,6 +95,15 @@ PS C:\> Mount-WindowsImage -Path "c:\offline" -Remount
 
 This command remounts a Windows image that has already been mounted to the c:\offline directory but has become inaccessible for servicing.
 
+### Example 5: Mount an image at an index of a file with logpath with checkintegrity with optimize
+```
```
Comment (illfated) [inline docset/windows/dism/Mount-WindowsImage.md]:
```suggestion
```PowerShell
```

File: docset/windows/dism/Mount-WindowsImage.md [hunk 2]
```diff
@@ -95,6 +95,15 @@ PS C:\> Mount-WindowsImage -Path "c:\offline" -Remount
...
+PS C:\> Mount-WindowsImage -Checkintegrity -ImagePath "c:\imagestore\install.wim" -Index 2 -Path "c:\offline" -Logpath C:\install.log -Optimize
+```
+
+This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with checkintegrity and getting log file with a quicker initial mount time.Additional operations may be slower.
```
Comment (illfated) [inline docset/windows/dism/Mount-WindowsImage.md]:
```suggestion
This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with Checkintegrity and getting a log file with a quicker initial mount time. Additional operations may be slower.
```

File: docset/windows/dism/Mount-WindowsImage.md [hunk 3]
```diff
@@ -95,6 +95,15 @@ PS C:\> Mount-WindowsImage -Path "c:\offline" -Remount
...
+
+
+
```
Comment (illfated) [inline docset/windows/dism/Mount-WindowsImage.md]:
```suggestion
```

**Comment by RAJU2529** [comment]
accepted

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@RAJU2529 Please re-target the target branch of the PR to `master` in order to proceed with the review. Thank you.

**Comment by e0i** [comment]
Hello, @RAJU2529, and @illfated Glad to see you on this repository, too! Would you re-target the PR branch to `master`? We would like to proceed with the review. Thanks.

**Comment by RAJU2529** [comment]
@e0i . how to change PR to master . Please give  steps

**Comment by RAJU2529** [comment]
@e0i. changed the PR from live to  master

**Comment by illfated** [comment]
Very good, well done. Please use the same procedure on your other powershell-docs PR.

**Comment by RAJU2529** [comment]
@illfaed. Ok sir. I will do

**Comment by RAJU2529** [comment]
@illfated . already changed PR live to master

**Comment by illfated** [comment]
Yes, I noticed. Thank you for your quick follow-up. 👍

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/dism/Mount-WindowsImage.md [hunk 4]
```diff
@@ -95,6 +95,14 @@ PS C:\> Mount-WindowsImage -Path "c:\offline" -Remount
 
 This command remounts a Windows image that has already been mounted to the c:\offline directory but has become inaccessible for servicing.
 
+### Example 5: Mount an image at an index of a file with logpath with checkintegrity with optimize
```
Comment (get-itips) [inline docset/windows/dism/Mount-WindowsImage.md]:
```suggestion
### Example 5: Mount an image at an index of a file specifying logpath with checkintegrity and optimize parameters
```

File: docset/windows/dism/Mount-WindowsImage.md [hunk 5]
```diff
@@ -95,6 +95,14 @@ PS C:\> Mount-WindowsImage -Path "c:\offline" -Remount
...
+```PowerShell
+PS C:\> Mount-WindowsImage -Checkintegrity -ImagePath "c:\imagestore\install.wim" -Index 2 -Path "c:\offline" -Logpath C:\install.log -Optimize
+```
+
+This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with Checkintegrity and getting a log file with a quicker initial mount time. Additional operations may be slower.
```
Comment (get-itips) [inline docset/windows/dism/Mount-WindowsImage.md]:
```suggestion
This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with Checkintegrity parameter, requesting a log file to be written and with a quicker initial mount time. Additional operations may be slower.
```

**Comment by illfated** [comment]
@get-itips : Could you add your Approval from the source page too (https://github.com/MicrosoftDocs/windows-powershell-docs/pull/1658/files) ?
image

**Comment by illfated** [comment]
#sign-off (yet again)

**Comment by e0i** [comment]
@RAJU2529 @illfated Thank you for your engagement, as always. Glad to see you here in this repository.

**Pull request merged**